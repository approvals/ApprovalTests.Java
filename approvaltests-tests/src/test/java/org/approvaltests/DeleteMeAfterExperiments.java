package org.approvaltests;

import org.junit.jupiter.api.Test;

class DeleteMeAfterExperiments
{
  @Test
  void applesauce()
  {
    Approvals.verify("test to see test report in github actions");
  }
}
