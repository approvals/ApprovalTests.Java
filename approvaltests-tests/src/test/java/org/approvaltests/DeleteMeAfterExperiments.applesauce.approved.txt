test to seee test report in github actions